app/models/project.rb

```diff
@@ -122,4 +122,14 @@ def project_events(order_by = "Completed IS NULL OR Completed = '0000-00-00', Co
       !p.completed && p.project_logs.empty? && p.date <= Date.today
     end
   end
+
+  def health_status_available?
+    if type == "Loan"
```

Comment:
Checking the type of an object in a conditional like this is usually an anti-pattern. The better way to do this is as follows:

1. In the parent class (Project), just return `false` in this method, very simple.
2. In the child class (Loan), override this method and return `!loan_health_check.nil?`.

Note also that `blank?` isn't needed in this case because blank only makes sense if the object defines `empty?`. Please see this: https://apidock.com/rails/Object/blank%3F.

Using blank? in the wrong place can lead to weird bugs.